AkkaHotSwap/src/main/java/org/ameya/App.java

```diff
@@ -18,6 +18,17 @@ public static void main(String args[]) throws  Exception{
         pingPongActor.tell(Messages.START,null);
         Thread.sleep(10000);
 
+        //the following code is to see whether codacy flags the obvious issues in this block
+        String first = "first";
+        String second = "second";
+
+        if(first == second){
+            System.err.println("Something went wrong");
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [System.err.println is used](https://www.codacy.com/app/ameya-kulkarni/akka-goodies/file/1684498547/issues/source?bid=3041792&fileBranchId=3041829#l26)